features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumSessionScope.kt

```diff
@@ -149,6 +152,7 @@ internal class RumSessionScope(
 
     private fun stopSession() {
         isActive = false
+        sessionEndedMetricDispatcher.onSessionStopped(sessionId)
```

Comment:
so we are going to call `onSessionStopped` only if `RumMonitor.stopSession` was called and we are not going to call it when session is stopped naturally (because of the expiration)?

Reply:
the purpose here is to register `was_stopped` in the metric when `stopSession` API is called by user, the metric will not be ended due to this. The metric is still ended when session has expired, timeout or stopped.